src/app/pages/DashboardPage/LearningMaterials.tsx

```diff
@@ -49,7 +49,12 @@ const getContent = (t: TFunction, layer: Layer) => {
         link: docs.sapphire,
         title: t('common.sapphire'),
       }
+    case Layer.cipher:
+      throw new AppError(AppErrors.UnsupportedLayer)
+    case Layer.consensus:
+      throw new AppError(AppErrors.UnsupportedLayer)
     default:
+      exhaustedTypeWarning('Unexpected layer', layer)
       throw new AppError(AppErrors.UnsupportedLayer)
```

Comment:
Here we are throwing two exceptions on quick succession. (At least in PROD mode.) Are they both necessary?

Reply:
Yes, unless we introduce more fallbacks